website/docs/api/admin/events-api.md

```diff
@@ -217,4 +145,1458 @@ interface IEvent {
+  "project": null,
+  "environment": null
+}
+```
+
+
+
+## Tag type events
+
+### `tag-type-created`
+
+This event fires when you create a new tag type.
+The `data` property contains the tag type configuration.
+
+``` json title="example event: tag-type-created"
+{
+  "id": 945,
+  "type": "tag-type-created",
+  "createdBy": "user@company.com",
+  "createdAt": "2022-06-02T13:27:01.235Z",
+  "data": {
+    "name": "new-tag-type",
+    "description": "event testing"
+  },
+  "preData": null,
+  "tags": [],
+  "featureName": null,
+  "project": null,
+  "environment": null
+}
+```
+
+### `tag-type-updated`
+
+This event fires when you update a tag type.
+The `data` property contains the new tag type configuration.
+
+``` json title="example event: tag-type-updated"
+{
+  "id": 946,
+  "type": "tag-type-updated",
+  "createdBy": "user@company.com",
+  "createdAt": "2022-06-02T13:27:31.126Z",
+  "data": {
+    "name": "new-tag-type",
+    "description": "This tag is for testing events."
+  },
+  "preData": null,
+  "tags": [],
+  "featureName": null,
+  "project": null,
+  "environment": null
+}
+```
+
+### `tag-type-deleted`
+
+This event fires when you delete a tag type.
+The `data` property contains the name of the deleted tag type.
+
+
+``` json title="example event: tag-type-deleted"
+{
+  "id": 947,
+  "type": "tag-type-deleted",
+  "createdBy": "user@company.com",
+  "createdAt": "2022-06-02T13:27:37.277Z",
+  "data": {
+    "name": "new-tag-type"
+  },
+  "preData": null,
+  "tags": [],
+  "featureName": null,
+  "project": null,
+  "environment": null
+}
+```
+
+### `tag-type-import`
+
+This event fires when you import a tag type as part of an import job. The `data` property contains the imported tag.
+
+``` json title="example event: tag-type-import"
+{
+  "id": 40,
+  "type": "tag-type-import",
+  "createdBy": "import-API-token",
+  "createdAt": "2022-06-03T11:30:40.599Z",
+  "data": {
+    "name": "custom-tag-type",
+    "description": "custom tag type",
+    "icon": null
+  },
+  "preData": null,
+  "tags": [],
+  "featureName": null,
+  "project": null,
+  "environment": null
+}
+```
+
+### `drop-tag-types`
+
+This event fires when you drop all existing tag types as part of a configuration import.
+
+``` json title="example event: drop-tag-types"
+{
+  "id": 38,
+  "type": "drop-tag-types",
+  "createdBy": "import-API-token",
+  "createdAt": "2022-06-03T11:30:40.596Z",
+  "data": {
+    "name": "all-tag-types"
+  },
+  "preData": null,
+  "tags": [],
+  "featureName": null,
+  "project": null,
+  "environment": null
+}
+```
+
+## Addon events
+
+### `addon-config-created`
+
+This event fires when you create an addon configuration.
+The `data` property contains the addon's ID and provider type.
+
+``` json title="example event: addon-config-created"
+{
+  "id": 960,
+  "type": "addon-config-created",
+  "createdBy": "user@company.com",
+  "createdAt": "2022-06-03T10:15:45.040Z",
+  "data": {
+    "provider": "webhook"
```

Comment:
The description above says there should be an addon ID here, should we include an example?

Reply:
Fixed in 2d85299c339ceb83513f8290ecfcb7fa46162f99